processing-service-core/global-config-options.md

```diff
@@ -0,0 +1,72 @@
+# Overview of global configuration options
+
+## Setting global configurations
+
+Parameters required to create or update global configuration:
```

Comment:
Can you put a link to this documentation in the architecture.md file so that someone reading the documentation can click through to it.